src/styles/blocks/header.scss

```diff
@@ -0,0 +1,69 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-inline: 50px;
+  box-shadow: 0 2px 4px 0 #0000000d;
+}
+
+.header__logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.header__logo-image {
+  width: 100%;
+  height: 100%;
+}
+
+.header__nav {
+  display: block;
+}
+
+.header__nav-list {
+  display: flex;
+  list-style: none;
+  gap: 20px;
+}
+
+.header__nav-link {
+  box-sizing: border-box;
+  position: relative;
+  height: 60px;
+  display: flex;
+  align-items: center;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  line-height: 14.06px;
+  font-weight: 500;
+  white-space: nowrap;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #060b35;
+  transition: color 0.3s;
+}
+
+.header__nav-link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
```

Comment:
Remove the extra space before the `:nth-child` selectors to ensure they work as intended. This aligns with the previous feedback regarding selector issues.